src/test/resources/moveRow.json

```diff
@@ -1 +1,13 @@
-{"resultCode":0,"result":[{"id":263016097834884,"rowNumber":1,"cells":[{"columnId":8764071660021636,"type":"TEXT_NUMBER","value":"New Data","displayValue":"New Data"}],"expanded":true,"createdAt":"2014-01-28T14:23:34-08:00","modifiedAt":"2014-01-28T14:49:25-08:00"}],"message":"SUCCESS"}
\ No newline at end of file
+{
```

Comment:
While unit tests with sample JSON files are good, be sure to run this thing against the actual api. We missed some bugs with v1 because it wasn't fully exercised against a real api.